### DISCUSSION TITLE
Arch Linux -git package problems updating to Kf5 name changes

### DISCUSSION THREAD

**Discussion by isf63** [lxqt/lxqt#2486] [Q&A] [answered]
I have most Qt5 LXQt components installed from the Arch Linux AUR, the exceptions being (not installed):
```
lxqt-openssh-askpass-git
lxqt-sudo-git
````

There are conflicts in some of these components when updating to 5.111.0 of the (newly renamed) `solid5` and `kwindowsystem5`.
```
:: installing kwindowsystem5 (5.111.0-1) breaks dependency 'kwindowsystem' required by lxqt-notificationd-git
:: installing solid5 (5.111.0-1) breaks dependency 'solid' required by lxqt-panel-git
:: installing kwindowsystem5 (5.111.0-1) breaks dependency 'kwindowsystem' required by lxqt-runner-git
:: installing kwindowsystem5 (5.111.0-1) breaks dependency 'kwindowsystem' required by lxqt-session-git
```

I thought this was due to the PKGBUILDs not being updated to the new name changes, but looking at:
- https://aur.archlinux.org/packages/lxqt-notificationd-git
- https://aur.archlinux.org/packages/lxqt-panel-git
- https://aur.archlinux.org/packages/lxqt-runner-git
- https://aur.archlinux.org/packages/lxqt-session-git

they all have kwindowsystem5/solid5.

I tried the logical `yay -S kwindowsystem5 solid5 lxqt-panel-git lxqt-notificationd-git lxqt-session-git lxqt-runner-git` which produced the same error as above. So as the PKGBUILDs have been updated, it is likely an issue with `pacman` / the AUR helper (`yay`).

##### System Information
* Distribution & Version: Arch Linux
* Package version: AUR -git

**Comment by tsujan** [comment]
This is a distro-specific question.

I don't include dependencies in my git LXQt PKGBUILDs and don't use `yay` but, having KDE here, I've seen similar cases (not related to LXQt) during upgrades of KDE packages a few times. For me the solution was `sudo pacman -Syu SOME_PACKAGE_NAMES`, where `SOME_PACKAGE_NAMES` were the names of those packages that caused the problem. Maybe a similar solution is possible with `yay`.

**Comment by stefonarch** [comment, marked as answer]
This morning I got similar errors
```
error: failed to prepare transaction (could not satisfy dependencies)
:: installing kwindowsystem5 (5.111.0-1) breaks dependency 'kwindowsystem' required by liblxqt-git
:: installing kwindowsystem5 (5.111.0-1) breaks dependency 'kwindowsystem' required by lxqt-notificationd-git
:: installing solid5 (5.111.0-1) breaks dependency 'solid' required by lxqt-panel-git
:: installing kidletime5 (5.111.0-1) breaks dependency 'kidletime' required by lxqt-powermanagement-git
:: installing solid5 (5.111.0-1) breaks dependency 'solid' required by lxqt-powermanagement-git
:: installing kwindowsystem5 (5.111.0-1) breaks dependency 'kwindowsystem' required by lxqt-runner-git
:: installing kwindowsystem5 (5.111.0-1) breaks dependency 'kwindowsystem' required by lxqt-session-git
:: installing kwindowsystem5 (5.111.0-1) breaks dependency 'kwindowsystem' required by screengrab-git

```

I've local PKGBUILDS and after adding the "5" where needed and recompiling them `yay -Syu` ran well. So probably first recompiling the AUR packages and only after that a full upgrade is working.

**Reply by isf63** [reply]
Thanks this pointed me in the right direction. Doing a clean-build and simultaneously upgrading (or alternatively removing the LXQt package momentarily) got it working for me.
